contracts/core/extensions/CoreIssuance.sol

```diff
@@ -56,6 +56,31 @@ contract CoreIssuance is
     {
         issueInternal(
             msg.sender,
+            msg.sender,
+            _set,
+            _quantity
+        );
+    }
+
+    /**
+     * Exchange components for Set tokens
+     *
+     * @param  _set          Address of the Set to issue
+     * @param  _quantity     Number of tokens to issue
+     * @param  _to           Address to issue to
+     */
+    function issueTo(
+        address _set,
+        uint256 _quantity,
+        address _to
```

Comment:
Only little naggy thing I'd say is that issueInternal has the _owner and _recipient first so for consistency sake make it first here to avoid the probable refactor we decide to do later.

Reply:
Good call